src/components/Settings/LogsSettings.tsx

```diff
@@ -45,14 +45,14 @@ export function LogsSettings() {
   return (
     <SettingsSection>
       <Flex mb="4" justify="between" align="center">
-        <Flex>
+        <Flex align="center">
           <Spinner mr="2" />
-          <Text size="2">
-            k6 Studio logs in this screen are updated in real-time.
+          <Text size="2" color="gray">
+            Application logs are updated in real-time.
           </Text>
         </Flex>
```

Comment:
I made it a bit more subtle, but I'm not quite happy with the spinner here: IMO it gives an indication that something is constantly fetching, even though it's not the case.